haskell-backend/src/main/java/org/kframework/backend/haskell/HaskellRewriter.java

```diff
@@ -294,8 +271,7 @@ private String saveKoreSpecToTemp(ModuleToKORE converter, Module rules) {
                 String koreOutput = converter.convertSpecificationModule(module, rules,
                         haskellKRunOptions.defaultClaimType, sb);
                 files.saveToTemp("spec.kore", koreOutput);
-                String specPath = files.resolveTemp("spec.kore").getAbsolutePath();
-                return specPath;
+                return files.resolveTemp("spec.kore").getAbsolutePath();
```

Comment:
I sometimes do this to put breakpoints and inspect the output of that function call.
Directly returning the result makes it difficult to inspect at runtime.
So, I wouldn't try to fix these in the future.

Reply:
This seems to be [supported in IntelliJ](https://stackoverflow.com/questions/5010362/can-i-find-out-the-return-value-before-returning-while-debugging-in-intellij); my preference is for the cleaner code in the common case at the expense of an extra step on the occasional time that this code needs run in a debugger, but that's not a strong preference.

Reply:
I wasn't aware of the feature. I guess we can remove them for now.